app/controllers/collections_controller.rb

```diff
@@ -84,7 +84,7 @@ def push_collection
   end
 
   def download_all
-    binary_zip_data = ProformaService::ExportTasks.call(tasks: @collection.tasks)
+    binary_zip_data = ProformaService::ExportTasks.call(tasks: @collection.tasks, options: {version: params[:version]})
```

Comment:
When manipulating the URL (in my browser) and specifying an unsupported version (such as `1.0`) an unhandled exception is thrown (`ProformaXML::PostGenerateValidationError "version not supported"`). Maybe we could handle this error, so that we don't respond with an internal server error in production?